modules/compile-only-dependencies.properties

```diff
@@ -0,0 +1,9 @@
+javax.servlet-api=javax.servlet:javax.servlet-api:3.0.1
+javax.ws.rs-api=javax.ws.rs:javax.ws.rs-api:2.1
+jstl=jstl:jstl:1.2
+osgi.cmpn=org.osgi:osgi.cmpn:6.0.0
+org.osgi.service.component.annotations=org.osgi:org.osgi.service.component.annotations:1.3.0
+org.osgi.service.jaxrs=org.osgi:org.osgi.service.jaxrs:1.0.0
+org.osgi.core=org.osgi:org.osgi.core:6.0.0
+portlet-api=javax.portlet:portlet-api:3.0.0
+release.portal.api=com.liferay.portal:release.portal.api:7.3.4-ga5
```

Comment:
@gamerson

Reply:
@CAustin582

Reply:
perhaps `bom-compile-only-dependencies.properties` would be a better name?